Update Documentation for 'Volumio' Media Player Component Configuration

vaidyasm opened the pull request:
**Description:**
Using new config vars
Related to Issue #6385

Pull request in [home-assistant](https://github.com/home-assistant/home-assistant): https://github.com/home-assistant/home-assistant.io/pull/7030

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

vaidyasm committed:
Update Documentation for 'Volumio' Media Player Component Configuration

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
+  required: false
+  type: string
+host:
+  description: IP address or hostname of the device
```
The IP [...].

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
+  required: true
+  type: string
+port:
+  description: Port number of Volumio service
```
The .

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
-- **port** (*Required*): Port number of Volumio service
+{% configuration %}
+name:
+  description: Name of the device
```
The name of the device.

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
+  description: Name of the device
+  required: false
+  type: string
+host:
```
Try to cross-check the documentation with the actual code.
In this case you can find the default values here: https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py#L49
`default: Volumio`

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
+  description: IP address or hostname of the device
+  required: true
+  type: string
+port:
```
Default value for host is localhost (https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py#L32)

tjorim commented on source/_components/media_player.volumio.markdown:
```diff
@@ -26,8 +26,17 @@ media_player:
+  description: Port number of Volumio service
+  required: true
+  type: integer
+{% endconfiguration %}
```
Same for port

tjorim reviewed (changes requested):
Thanks for helping us out with this PR! I left some feedback.

vaidyasm committed:
Update description sentence style.

vaidyasm committed:
Adds default values

Default values are obtained from https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py lines 32 through 34.

vaidyasm commented:
@tjorim Thanks for your feedbacks. They have been incorporated. It helps to know source where the defaults were defined.

frenck commented:
Rebased this PR onto `current`.
